myhw-week2/database.js

```diff
@@ -0,0 +1,73 @@
+const mysql = require('mysql');
+const util = require('util');
+const connection = mysql.createConnection({
+  host: 'localhost',
+  user: 'hyfuser',
+  password: 'hyfpassword',
+  database: 'new_world'
+});
+
+
+const execQuery = util.promisify(connection.query.bind(connection));
+
+async function seedDatabase() {
+
+
+  //Part 1 - 1
+  let country_name = 'Syria';
+  let capitalOfCountry = 'select country.name, city.name from city inner join country on city.id = country.capital where country.name = ' + connection.escape(country_name);
+
+  //Part 1 - 2
+  let country_region = 'Western Europe';
+  let findingSpokenLanguages = 'select countryLanguage.language, country.region from country join countryLanguage on country.Code = countryLanguage.countryCode where country.region =' + connection.escape(country_region);
```

Comment:
One way to write less code is to give your tables alias, for example `countryLanguage as l` and then you can join on `l.countryCode` ;)